### DISCUSSION TITLE
Company website for the device

### DISCUSSION THREAD

**Discussion by parabyte** [witnessmenow/ESP32-Cheap-Yellow-Display#173] [General]
I managed to find the website for the device, if you google the filenames you can find schematics and all sorts of infomation! 

http://www.jczn1688.com/zlxz

**Comment by parabyte** [comment]
found this too https://macsbug.wordpress.com/2022/08/17/esp32-2432s028/

**Reply by parabyte** [reply]
hope this helps

**Reply by aly-fly** [reply]
Translated from Japanese.
[ESP32-2432S028_macsbug.pdf](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/files/14969885/ESP32-2432S028_macsbug.pdf)

**Comment by Xylopyrographer** [comment]
Did you find the magic user and password to allow downloads from the jczn1699.com site?

**Reply by parabyte** [reply]
unfortunately  no, but if you google the filenames you will find the files, i will ask my chinese speaking friend if he can figure it out :)
